lxd/devlxd.go

```diff
@@ -103,6 +103,14 @@ func devlxdConfigKeyGetHandler(d *Daemon, c instance.Instance, w http.ResponseWr
 		return response.DevLxdErrorResponse(api.StatusErrorf(http.StatusNotFound, "not found"), c.Type() == instancetype.VM)
 	}
 
+	// If parsing the config is not possible, abstain from merge the additional keys into the config.
+	if strings.HasSuffix(key, ".vendor-data") || strings.HasSuffix(key, ".user-data") {
+		value, err = mergeSSHKeyCloudConfig(c.ExpandedConfig(), value)
+		if err != nil {
+			logger.Warn("Failed merging SSH keys into cloud config, abstain from injecting additional keys")
```

Comment:
Lets log the request key here too please

Reply:
we also should include `instance` and `project` in the logging context as we have the instance available.